src/org/scada_lts/web/mvc/api/DataPointAPI.java

```diff
@@ -22,6 +22,7 @@
 import com.serotonin.mango.vo.DataPointVO;
 import com.serotonin.mango.vo.User;
 import com.serotonin.mango.web.dwr.EmportDwr;
+import com.serotonin.mango.web.dwr.beans.DataPointBean;
```

Comment:
Why are you importing something somewhere where you have not provided any changes? Was there any error caused by missing import?